aten/src/ATen/native/Copy.cpp

```diff
@@ -10,11 +10,27 @@ namespace {
 
 template <typename self_T, typename src_T>
 void _copy__cpu(at::Tensor& self, const at::Tensor& src) {
-  at::CPU_tensor_apply2<self_T, src_T>(
-      self, src, [](self_T& self_val, const src_T& src_val) {
-        self_val = static_cast<self_T>(
+  bool serial_path = false;
+#ifdef _OPENMP
+  if (!at::in_parallel_region()) {
+    at::CPU_tensor_parallel_apply2<self_T, src_T>(
+        self, src, [](self_T& self_val, const src_T& src_val) {
+          self_val = static_cast<self_T>(
             static_cast<at::native::inter_copy_type_t<self_T>>(src_val));
-      });
+          }, 20000);
+  } else {
+    serial_path = true;
+  }
+#else
+  serial_path = true;
```

Comment:
nit: this can be simplified by writint it out as
```cpp
#ifdef _OPENMP
  if (!at::in_prallel_region()) {
    <parallel path here>
    return;
  }
#endif
  <serial path here>
```

Reply:
Ok, I think it will be better if we also change the code here：https://github.com/pytorch/pytorch/blob/3fcee4875c4383bff78917284875f180ad41903c/aten/src/ATen/native/Copy.cpp#L116.  which will have a same code style. I will pull a new commit if you think it is ok. Thanks!